### ISSUE TITLE
"stupid" question

### ISSUE THREAD

**Issue by txmamash** [josephwalden13/tweak-installer#29]
how to install it at first then how to install the tweak is there a guidline for no computer method?
thanks.

**Comment by StalkerSea** [comment]
There's no "Install", you just unzip it, and open the "Tweak Installer.exe". (You need to keep in mind that your computer and your iDevice must be connected on the same Wi-Fi) 

After opening "Tweak Installer.exe", you need to go to your iDevice, open Settings, go to Wi-Fi, and press on the little "i" on your connected Wi-Fi, then you write the address that's on "IP Address" on the field that's on the "Tweak Installer.exe" naming "Device IP", for the "root password" field, you type "alpine", then on "port" you put "22" or "2222". After you've done all of this, you need to select a ".deb" file (basically, Cydia tweaks) in "Select Debs, Zips and IPAs". You can select multiple .debs at the same time if you want, but I'd advise against it, since a "timeout" can happen (basically, a crash on the install, It has happened to me some times, but not that many). When you're done selecting the .debs you want to install, you click on the big button that says "Install", press "Yes" on any prompts regarding "tic.exe" and "7z.exe", and wait it for finish the install. 

If you want to uninstall any ".deb" you installed before, for some reason (maybe you don't want it anymore), just do the same process as before, but instead of clicking on "Install", you click on "Uninstall".

You can also install "Filza" fairly easy (file viewer for your iDevice), you just need to right click on "Install", and there will be an option saying "Install Filza", same for "Uninstall".


And no, this is just for Windows, for this program there is no "no computer method".

If you want to install a ".deb" without using a computer, you can't use this executable, and you must follow a series of steps, that make you come back to this installer. But if you insist on installing tweaks without using a computer, you can search up an online tutorial that helps you on this.

Basically you unzip the ".deb", unzip "data.tar", and move everything to their respective folders to your iDevice.

I've seen a few of tutorials, but those are just too confusing, and you can put your iDevice in a bootloop if you delete something you weren't intended to, so I'd say that you should stay with this installer.


Hope this is of any help!

**Comment by LuckyAzumi** [comment]
And no, this is just for Windows, for this program there is no "no computer method".

**Comment by txmamash** [comment]
thank you so much @LuckyAzumi
that was a great help.
really appreciate it.
